net/remote/remote-client.sh

```diff
@@ -68,25 +69,32 @@ solana-bench-tps)
   net/scripts/rsync-retry.sh -vPrc \
     "$entrypointIp":~/solana/config/bench-tps"$clientIndex".yml ./client-accounts.yml
 
+  net/scripts/rsync-retry.sh -vPrc \
+    "$entrypointIp":~/solana/config/validator-identity-1.json ./validator-identity.json
+
   args=()
 
   if ${TPU_CLIENT}; then
     args+=(--use-tpu-client)
-    args+=(--url "http://$entrypointIp:8899")
   elif ${RPC_CLIENT}; then
     args+=(--use-rpc-client)
-    args+=(--url "http://$entrypointIp:8899")
   else
     args+=(--entrypoint "$entrypointIp:8001")
   fi
 
+  if [[ -z "$maybeUseUnstakedConnection" ]]; then
+    args+=(--bind-address "$entrypointIp")
+    args+=(--client-node-id ./validator-identity.json)
+  fi
+
   clientCommand="\
     solana-bench-tps \
       --duration 7500 \
       --sustained \
       --threads $threadCount \
       $benchTpsExtraArgs \
       --read-client-keys ./client-accounts.yml \
+      --url "http://$entrypointIp:8899" \
```

Comment:
In case if we use staked connection, we need to specify rpc address even if ThinClient is employed. This address is used to create RpcClient to check stake of the specified address and later another RpcClient is created. So I believe that there is no harm in specifying it always